src/utils/rules-proxy.js

```diff
@@ -55,11 +60,11 @@ const createRewriter = async function ({ distDir, projectDir, jwtSecret, jwtRole
   let matcher = null
   const configFiles = [
     ...new Set(
-      [path.resolve(distDir, '_redirects'), path.resolve(projectDir, '_redirects')].concat(
-        configPath ? path.resolve(configPath) : [],
+      [path.resolve(distDir, '_redirects'), path.resolve(projectDir, '_redirects'), configPath].filter(
```

Comment:
@ehmicky `configPath` can be `undefined` so `path.resolve(configPath)` failed in some of the tests.
I believe `@netlify/config` returns the resolved path if a config file exists so we can remove the call to `path.resolve`.
Let me know if I'm correct on this.

Reply:
Yes, it is an absolute path :+1: